src/unix/core.c

```diff
@@ -462,7 +472,7 @@ int uv__accept(int sockfd) {
   assert(sockfd >= 0);
 
   while (1) {
-#if defined(__linux__) || (defined(__FreeBSD__) && __FreeBSD__ >= 10)
+#if defined(__linux__) || (defined(__FreeBSD__) && __FreeBSD__ >= 10) || defined(__NetBSD__)
```

Comment:
This line is over 80 characters now.

Reply:
break it up with \ at the end and the new statement in a new line?